Site updates moving from RHElements to PFElements (#17)
RHElements/rhelements-documentation: content/develop/step-2a.md

PFElement.create(PfeCoolElement);
```

> There are a couple of handy methods that have been commented out that are not as frequently used but are there for your convenience. If you're not using the connectedCallback, observedAttributes, disconnectedCallback, or attributeChangedCallback methods, you can just delete them.

Reviewer: ```suggestion
There are a couple of handy methods (commented out) which are not as frequently used but are there for your convenience. If you're not using the connectedCallback, observedAttributes, disconnectedCallback, or attributeChangedCallback methods, you can just delete them.
```
